worker/include/RTC/RtpStreamSend.hpp

```diff
@@ -20,10 +20,14 @@ namespace RTC
 	public:
 		struct StorageItem
 		{
+			// Original packet.
+			RTC::RtpPacket::SharedPtr originalPacket{ nullptr };
+			// Correct SSRC since original packet will have original ssrc.
+			uint32_t ssrc{ 0 };
+			// Correct sequence number since original packet will have original sequence number.
+			uint16_t sequenceNumber{ 0 };
 			// Cloned packet.
-			RTC::RtpPacket* packet{ nullptr };
-			// Memory to hold the cloned packet (with extra space for RTX encoding).
-			uint8_t store[RTC::MtuSize + 100];
+			RTC::RtpPacket::SharedPtr clonedPacket{ nullptr };
```

Comment:
This represents the RtpPacket that is re-sent upon receiving the corresponding NACK.

**Each** RtpStreamSend re-sending this packet creates its own clone of the original RTP packet, modifies the SSRC and Cseq, or creates the corresponding RTX packet if needed.

Instead of doing that a lot of memory would be saved if the RTX packet of the original was shared among all the `RtpStreamSend`  instances as it is done with `originalPacket`. Then, before re-sending the packet in `ReceiveNack`, the `originalPacket`s or `clonnedPacket`s (depending on whether RTX is being used) SSRC and Cseq needs to be rewriteen to fit the one of this RtpStreamSend, and later restored.

Following the above rationale `clonedPacket` should be renamed to `rtxPacket`. 
Once at this point, `originalPacket` (which is also a cloned packet, BTW) should be IMO better named to just `packet`.

Reply:
Resolving the comment. This will be done later.